src/utils.ts

```diff
@@ -71,8 +65,8 @@ function printTable(alignment, lines) {
   // Now that we know how wide each column is, go and print them.
   for (const line of lines) {
     let lineStr = '';
-    for (idx in width) {
-      if (idx > 0) {
+    for (const idx in width) {
+      if (<number > (<unknown>idx) > 0) {
```

Comment:
nitpick: extra space

```suggestion
      if (<number> (<unknown>idx) > 0) {
```

Reply:
Damn, seems like the eslint formatter didn't see it.

Reply:
That was actually the eslint formatter itself :grimacing: 
I fixed it